doc/manual/manual.tex

```diff
@@ -7860,12 +7856,18 @@ \subsubsection{Inner core convection}
   H,
 \end{align}
 
-where $Ra$ is the Rayleigh number. 
+where $Ra$ is the Rayleigh number and 6 is the 'source term', originated from the time-dependence of the reference temperature (secular cooling). In spherical geometry, $H=6$. 
+
+\vspace{0.3cm}
+\textbf{Thermal evolution of the inner core.}
+Thermal evolution of the inner core implies that, owing to secular cooling, the temperature decreases with respect to the initial state. The growing inner core is however stable if formerly solidified material is colder than the current adiabatic profil attached to the liquidus temperature at the inner core boundary, as a result of diffusion. If not, it is unstable to thermal convection.  
```

Comment:
profil --> profile

Reply:
I don't really understand the part about the stable/unstable conditions, in particular the relation to the liquidus temperature. Do you mean that the adiabatic temperature always starts from the liquidus temperature at the inner core boundary? It could make sense to put that part into a separate sentence (that comes earlier).

Reply:
I removed it and added a sentence before!